### PR TITLE
feat: add samples for numeric

### PR DESCRIPTION
**Pull request by olavloite**
Adds samples for the `NUMERIC` data type to the existing JDBC samples.

### CODE REVIEW

File: spanner/jdbc/src/main/java/com/example/spanner/jdbc/AutocommitUpdateDataExample.java [hunk 1]
```diff
@@ -45,8 +45,8 @@ static void update(String projectId, String instanceId, String databaseId) throw
       // The following statement will automatically be committed by the connection.
       int updateCount =
           statement.executeUpdate(
-              "INSERT INTO Singers (SingerId, FirstName, LastName)\n"
-                  + "VALUES (9999, 'Lloyd', 'Pineda')");
+              "INSERT INTO Singers (SingerId, FirstName, LastName, Revenues)\n"
```
Comment (lesv) [inline spanner/jdbc/src/main/java/com/example/spanner/jdbc/AutocommitUpdateDataExample.java:48]: Why do you add the \n ?

**Review by lesv — APPROVED** [review]
LGTM - I don't understand why you are adding \n everywhere at the end of strings you are concatenating.  It seems superfluous.

**Review by skuruppu — APPROVED** [review]

File: spanner/jdbc/src/main/java/com/example/spanner/jdbc/AutocommitUpdateDataExample.java (hunk 1 above)
Comment (olavloite) [reply spanner/jdbc/src/main/java/com/example/spanner/jdbc/AutocommitUpdateDataExample.java:48]: (It's not specifically added in this change, so I assume that it is more of a general question for all these samples)

No really specific reason, other than that it is quite customary to have the `VALUES (...)` clause on a new line. See for example the examples here: https://cloud.google.com/spanner/docs/dml-syntax#insert_examples

**Comment by olavloite** [comment]
@lesv @skuruppu I'm apparently not authorized to merge these changes (anymore?). Would one of you do that?

**Comment by lesv** [comment]
I merged, but I still don't understand why so many `\n`'s are present in the SQL

**Pull request merged**